l10n_it_fatturapa_out/wizard/efattura.py

```diff
@@ -110,7 +110,7 @@ def format_quantity(line):
         def get_id_fiscale_iva(partner, prefer_fiscalcode=False):
             id_paese = partner.country_id.code
             if partner.vat:
-                if id_paese == "IT" and partner.vat.startswith("IT"):
+                if partner.vat.startswith(id_paese) and not id_paese == "CH":
```

Comment:
È un'assuzione sbagliata che per tutti i paesi del mondo sia possible rimuovere il prefisso, se coincide col codice ISO del paese.

Inoltre è inutile perché lo SdI controlla solo in caso di IT.

(per dettagli vedi l'altro mio commento)

Reply:
Abbiamo il caso, in cui un cliente fa fattura verso l'austria e avendo due volte ATAT<vat> nell'id_fiscale viene scartata

Reply:
RIguardo lo slicing l'ho preso un po' alla leggera e posso correggerlo, ma l'id_fiscale deve essere composto da id_paese e vat senza rimettere il valore di id_paese nel codice del partner

Reply:
> senza rimettere il valore di id_paese nel codice

Hai un riferimento soprattutto per questo? la documentazione SdI menziona solo IT come caso particolare. 

Hai visto quello che ho scritto per Grecia e Irlanda del Nord?  Non è sempre vero che il prefisso è l'IdPaese. Per la grecia IdPaese = GR e il codice inizia per EL. Per l'IdN, IdPaese = GB, il codice inizia per XI.

La scelta è stata quella di lasciare intatto il campo, senza rimuovere o aggiungere nulla (come veniva fatto in passato). La piva indicata nel partner viene copiata così com'è, a meno che non sia IT.

Va inserita correttamente (col prefisso se serve).

> Abbiamo il caso, in cui un cliente fa fattura verso l'austria e avendo due volte ATAT nell'id_fiscale viene scartata
il codice attuale non aggiunge AT davanti alla piva. Qual è il codice di errore SdI?